### PR TITLE
[src] getPolicyAction

### PR DESCRIPTION
**Pull request by Lironrad**

### CODE REVIEW

File: api/services/administration/general/paths/getPolicyAction.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+get:
+  summary: "Get Access Policy Actions"
+  description: Get actions for access policies.
+  operationId: "OrganizationsAndAccountsGetAccessPolicyAction"
+  tags:
+    - "Organizations"
+  parameters:
```
Comment (sagishoshan) [inline api/services/administration/general/paths/getPolicyAction.yaml:7]: lets move all parameters to: `api/services/administration/general/parameters` as it is only relevant to policy action

File: api/services/administration/general/responses/getPolicyAction.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
+description: Get actions for access policies Response
+content:
+  application/json:
+    schema:
+      allOf:
+        - allOf:
```
Comment (sagishoshan) [inline api/services/administration/general/responses/getPolicyAction.yaml:6]: schema should inherit from `"../../../../../commons/schemas/responseItemWrapper.yaml"`.
only example should appear here.

you can take an example from `services/administration/general/responses/userGroup/getUserGroups.yaml`

File: api/services/administration/general/responses/getPolicyAction.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                    kind:
+                      type: "string"
+                      example: spotinst:setup:policyAction
+                      description: >
+                        The data type of each item in the response
+                    items:
+                      type: "array"
+                      description: >
+                        Array of data objects
+                      items:
```
Comment (sagishoshan) [inline api/services/administration/general/responses/getPolicyAction.yaml:66]: no need to create different example files.
you can add example to the schema file itself. 

you can take an example from `/services/administration/general/schemas/userGroup/getUserGroups.yaml`

**Review by sagishoshan — CHANGES_REQUESTED** [review]

File: api/services/administration/general/paths/getPolicyAction.yaml (hunk 1 above)
Comment (Lironrad) [reply api/services/administration/general/paths/getPolicyAction.yaml:7]: done

File: api/services/administration/general/responses/getPolicyAction.yaml (hunk 2 above)
Comment (Lironrad) [reply api/services/administration/general/responses/getPolicyAction.yaml:6]: done

File: api/services/administration/general/responses/getPolicyAction.yaml (hunk 3 above)
Comment (Lironrad) [reply api/services/administration/general/responses/getPolicyAction.yaml:66]: done

**Review by sagishoshan — APPROVED** [review]

**Pull request merged**